spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,83 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    before do
```

Comment:
`before`s, like `let`s, are best for details which are necessary but not relevant to understanding the behavior being exercised. Since both these objects are central, I'd resist the urge to DRY up here and allow a little duplication for the sake of readability